libraries/botbuilder-dialogs-adaptive-teams/package.json

```diff
@@ -34,6 +34,7 @@
     "botbuilder-dialogs-adaptive": "4.1.6",
     "botbuilder-dialogs-declarative": "4.1.6",
     "botbuilder-stdlib": "4.1.6",
+    "botframework-schema": "4.1.6",
```

Comment:
The schema files are all exported via `botbuilder` so you shouldn't need the explicit `botbuilder-schema` reference.